app/views/layouts/_footer.html.erb

```diff
@@ -34,6 +34,9 @@
         <li class="govuk-footer__inline-list-item">
           <%= govuk_link_to("Privacy notice", privacy_notice_path, class: "govuk-footer__link") %>
         </li>
+        <li class="govuk-footer__inline-list-item">
```

Comment:
No biggie, but I would list these in alphabetical order, so accessibility statement comes first in this list.